code/src/components/NameInput.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+export const NameInput = ({nameInput, nameUpdateInput, stepChange}) => {
+
+
+   
```

Comment:
So you have used the form tags in the components, but you could also make the onSubmit  (instead of onClick) funtionality on them also.